File: leetcode.com/2490_circular_sentence/solution.py
class Solution:
    def isCircularSentence(self, sentence: str) -> bool:
        a = sentence.split()

        for i in range(len(a) - 1):
            if a[i][-1] != a[i + 1][0]:
                return False

        if a[-1][-1] != a[0][0]:
            return False

        return True

File: leetcode.com/2490_circular_sentence/solution.go
func isCircularSentence(sentence string) bool {
    if sentence[0] != sentence[len(sentence) - 1] {
        return false
    }

    for i := 0 ; i < len(sentence) ; i++ {
        if sentence[i] == ' ' && sentence[i - 1] != sentence[i + 1] {
            return false
        }
    }

    return true
}
